packages/eslint-plugin/src/rules/no-tslint-comments.js

```diff
@@ -17,7 +17,7 @@ module.exports = createRule({
   defaultOptions: [],
   create: context => {
     const tslintCommentRegex = /^\s*tslint ?: ?(disable|enable)/;
-    const sourceCode = context.getSourceCode();
```

Comment:
[getSourceCode](https://github.com/typescript-eslint/typescript-eslint/pull/8377/files#diff-a916a57c423a9435766d31cbea70b6434185d7f227eaff16df942efaab05b08bL37) is deprecated in favour of `sourceCode`